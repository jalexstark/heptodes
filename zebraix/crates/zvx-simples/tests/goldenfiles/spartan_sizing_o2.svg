<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<defs>
<g>
<g id="glyph-0-0">
<path d="M 0.375 -2.140625 L 0.375 -2.875 L 2.640625 -2.875 L 2.640625 -2.140625 Z M 0.375 -2.140625 "/>
</g>
<g id="glyph-0-1">
<path d="M 5.15625 -1.515625 L 4.1875 -1.515625 L 4.1875 0 L 3.390625 0 L 3.390625 -1.515625 L 0.203125 -1.515625 L 0.203125 -2.21875 L 3.328125 -6.703125 L 4.1875 -6.703125 L 4.1875 -2.25 L 5.15625 -2.25 Z M 3.390625 -4.359375 C 3.390625 -4.679688 3.394531 -4.953125 3.40625 -5.171875 C 3.414062 -5.398438 3.425781 -5.617188 3.4375 -5.828125 L 3.40625 -5.828125 C 3.351562 -5.710938 3.289062 -5.582031 3.21875 -5.4375 C 3.144531 -5.300781 3.070312 -5.191406 3 -5.109375 L 1 -2.25 L 3.390625 -2.25 Z M 3.390625 -4.359375 "/>
</g>
<g id="glyph-0-2">
<path d="M 0.671875 -0.5 C 0.671875 -0.726562 0.726562 -0.890625 0.84375 -0.984375 C 0.957031 -1.078125 1.09375 -1.125 1.25 -1.125 C 1.40625 -1.125 1.539062 -1.078125 1.65625 -0.984375 C 1.769531 -0.890625 1.828125 -0.726562 1.828125 -0.5 C 1.828125 -0.28125 1.769531 -0.117188 1.65625 -0.015625 C 1.539062 0.078125 1.40625 0.125 1.25 0.125 C 1.09375 0.125 0.957031 0.078125 0.84375 -0.015625 C 0.726562 -0.117188 0.671875 -0.28125 0.671875 -0.5 Z M 0.671875 -0.5 "/>
</g>
<g id="glyph-0-3">
<path d="M 4.890625 -3.34375 C 4.890625 -2.625 4.8125 -2.007812 4.65625 -1.5 C 4.507812 -0.988281 4.273438 -0.59375 3.953125 -0.3125 C 3.628906 -0.0390625 3.195312 0.09375 2.65625 0.09375 C 1.90625 0.09375 1.347656 -0.207031 0.984375 -0.8125 C 0.628906 -1.425781 0.453125 -2.269531 0.453125 -3.34375 C 0.453125 -4.0625 0.523438 -4.675781 0.671875 -5.1875 C 0.816406 -5.707031 1.050781 -6.101562 1.375 -6.375 C 1.707031 -6.644531 2.132812 -6.78125 2.65625 -6.78125 C 3.40625 -6.78125 3.960938 -6.476562 4.328125 -5.875 C 4.703125 -5.269531 4.890625 -4.425781 4.890625 -3.34375 Z M 1.28125 -3.34375 C 1.28125 -2.425781 1.382812 -1.738281 1.59375 -1.28125 C 1.800781 -0.832031 2.15625 -0.609375 2.65625 -0.609375 C 3.15625 -0.609375 3.515625 -0.832031 3.734375 -1.28125 C 3.953125 -1.738281 4.0625 -2.425781 4.0625 -3.34375 C 4.0625 -4.25 3.953125 -4.929688 3.734375 -5.390625 C 3.515625 -5.847656 3.15625 -6.078125 2.65625 -6.078125 C 2.15625 -6.078125 1.800781 -5.847656 1.59375 -5.390625 C 1.382812 -4.929688 1.28125 -4.25 1.28125 -3.34375 Z M 1.28125 -3.34375 "/>
</g>
<g id="glyph-0-4">
<path d="M 3.3125 0 L 2.515625 0 L 2.515625 -4.65625 C 2.515625 -4.925781 2.515625 -5.140625 2.515625 -5.296875 C 2.523438 -5.460938 2.535156 -5.632812 2.546875 -5.8125 C 2.453125 -5.707031 2.363281 -5.617188 2.28125 -5.546875 C 2.195312 -5.484375 2.09375 -5.398438 1.96875 -5.296875 L 1.265625 -4.71875 L 0.828125 -5.265625 L 2.640625 -6.671875 L 3.3125 -6.671875 Z M 3.3125 0 "/>
</g>
<g id="glyph-0-5">
<path d="M 4.859375 0 L 0.453125 0 L 0.453125 -0.6875 L 2.203125 -2.453125 C 2.535156 -2.785156 2.816406 -3.082031 3.046875 -3.34375 C 3.273438 -3.601562 3.445312 -3.859375 3.5625 -4.109375 C 3.6875 -4.367188 3.75 -4.644531 3.75 -4.9375 C 3.75 -5.3125 3.640625 -5.59375 3.421875 -5.78125 C 3.203125 -5.96875 2.914062 -6.0625 2.5625 -6.0625 C 2.238281 -6.0625 1.953125 -6.003906 1.703125 -5.890625 C 1.460938 -5.785156 1.21875 -5.628906 0.96875 -5.421875 L 0.515625 -5.984375 C 0.785156 -6.203125 1.085938 -6.382812 1.421875 -6.53125 C 1.765625 -6.6875 2.144531 -6.765625 2.5625 -6.765625 C 3.1875 -6.765625 3.679688 -6.609375 4.046875 -6.296875 C 4.410156 -5.984375 4.59375 -5.546875 4.59375 -4.984375 C 4.59375 -4.640625 4.519531 -4.3125 4.375 -4 C 4.226562 -3.695312 4.023438 -3.394531 3.765625 -3.09375 C 3.515625 -2.800781 3.21875 -2.488281 2.875 -2.15625 L 1.484375 -0.78125 L 1.484375 -0.75 L 4.859375 -0.75 Z M 4.859375 0 "/>
</g>
</g>
</defs>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 18.183594 286.363281 L 18.183594 13.636719 L 381.816406 13.636719 L 381.816406 286.363281 Z M 18.183594 286.363281 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 18.183594 150 L 381.816406 150 M 18.183594 204.546875 L 381.816406 204.546875 M 18.183594 259.089844 L 381.816406 259.089844 M 18.183594 95.453125 L 381.816406 95.453125 M 18.183594 40.910156 L 381.816406 40.910156 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 290.910156 286.363281 L 290.910156 13.636719 M 230.304688 286.363281 L 230.304688 13.636719 M 169.695312 286.363281 L 169.695312 13.636719 M 109.089844 286.363281 L 109.089844 13.636719 M 48.484375 286.363281 L 48.484375 13.636719 M 351.515625 286.363281 L 351.515625 13.636719 "/>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="40.484375" y="298.229492"/>
<use xlink:href="#glyph-0-1" x="43.484375" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="48.484375" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="51.484375" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="284.410156" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="289.410156" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="292.410156" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-4" x="345.015625" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="350.015625" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="353.015625" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="-2.492188" y="261.866211"/>
<use xlink:href="#glyph-0-5" x="0.507812" y="261.866211"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="261.866211"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="261.866211"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="0.507812" y="152.776367"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="152.776367"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="152.776367"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-5" x="0.507812" y="43.686523"/>
<use xlink:href="#glyph-0-2" x="5.507812" y="43.686523"/>
<use xlink:href="#glyph-0-3" x="8.507812" y="43.686523"/>
</g>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 168.765625 83.882812 C 168.765625 87.617188 163.167969 87.617188 163.167969 83.882812 C 163.167969 80.152344 168.765625 80.152344 168.765625 83.882812 Z M 182.230469 78.96875 C 182.230469 82.699219 176.628906 82.699219 176.628906 78.96875 C 176.628906 75.234375 182.230469 75.234375 182.230469 78.96875 Z M 201.414062 74.191406 C 201.414062 77.925781 195.8125 77.925781 195.8125 74.191406 C 195.8125 70.457031 201.414062 70.457031 201.414062 74.191406 Z M 229.140625 71.890625 C 229.140625 75.625 223.542969 75.625 223.542969 71.890625 C 223.542969 68.15625 229.140625 68.15625 229.140625 71.890625 Z M 266.882812 79.441406 C 266.882812 83.175781 261.285156 83.175781 261.285156 79.441406 C 261.285156 75.707031 266.882812 75.707031 266.882812 79.441406 Z M 301.746094 114.171875 C 301.746094 117.90625 296.144531 117.90625 296.144531 114.171875 C 296.144531 110.4375 301.746094 110.4375 301.746094 114.171875 Z M 285.890625 183.449219 C 285.890625 187.183594 280.292969 187.183594 280.292969 183.449219 C 280.292969 179.71875 285.890625 179.71875 285.890625 183.449219 Z M 209.683594 234.527344 C 209.683594 238.257812 204.085938 238.257812 204.085938 234.527344 C 204.085938 230.792969 209.683594 230.792969 209.683594 234.527344 Z M 144.722656 239.148438 C 144.722656 242.878906 139.125 242.878906 139.125 239.148438 C 139.125 235.414062 144.722656 235.414062 144.722656 239.148438 Z M 111.746094 225.367188 C 111.746094 229.101562 106.144531 229.101562 106.144531 225.367188 C 106.144531 221.636719 111.746094 221.636719 111.746094 225.367188 Z M 97.273438 210.339844 C 97.273438 214.070312 91.675781 214.070312 91.675781 210.339844 C 91.675781 206.605469 97.273438 206.605469 97.273438 210.339844 Z M 91.21875 197.949219 C 91.21875 201.683594 85.621094 201.683594 85.621094 197.949219 C 85.621094 194.21875 91.21875 194.21875 91.21875 197.949219 Z M 88.914062 188.277344 C 88.914062 192.011719 83.316406 192.011719 83.316406 188.277344 C 83.316406 184.542969 88.914062 184.542969 88.914062 188.277344 Z M 88.914062 188.277344 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(60%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 165.964844 83.882812 L 168.796875 82.738281 L 171.863281 81.566406 L 175.1875 80.375 L 178.796875 79.167969 L 182.730469 77.964844 L 187.015625 76.773438 L 191.691406 75.621094 L 196.804688 74.53125 L 202.394531 73.550781 L 208.511719 72.730469 L 215.1875 72.140625 L 222.46875 71.867188 L 230.371094 72.039062 L 238.894531 72.8125 L 247.992188 74.375 L 257.550781 76.984375 L 267.367188 80.921875 L 277.097656 86.527344 L 286.230469 94.132812 L 294.054688 104.039062 L 299.6875 116.402344 L 302.140625 131.144531 L 300.496094 147.835938 L 294.164062 165.648438 L 283.089844 183.449219 L 267.882812 200.007812 L 249.699219 214.246094 L 229.980469 225.480469 L 210.125 233.496094 L 191.234375 238.464844 L 174.03125 240.820312 L 158.863281 241.097656 L 145.816406 239.832031 L 134.785156 237.480469 L 125.570312 234.425781 L 117.9375 230.945312 L 111.652344 227.25 L 106.5 223.484375 L 102.285156 219.746094 L 98.84375 216.101562 L 96.042969 212.59375 L 93.765625 209.238281 L 91.921875 206.054688 L 90.433594 203.039062 L 89.234375 200.195312 L 88.273438 197.515625 L 87.515625 194.992188 L 86.917969 192.617188 L 86.460938 190.382812 L 86.113281 188.277344 "/>
</svg>
