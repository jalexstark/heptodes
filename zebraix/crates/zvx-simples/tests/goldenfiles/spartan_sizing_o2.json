{
  "sizing_scheme": "Fill",
  "axes_range": [
    -4.5,
    -2.5,
    1.5,
    2.5
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "segments": [
            {
              "Polyline": [
                [
                  -4.5,
                  -2.5
                ],
                [
                  -4.5,
                  2.5
                ],
                [
                  1.5,
                  2.5
                ],
                [
                  1.5,
                  -2.5
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -4.5,
                  0.0
                ],
                [
                  1.5,
                  0.0
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                -0.0,
                -1.0
              ],
              [
                -0.0,
                -2.0
              ],
              [
                0.0,
                1.0
              ],
              [
                0.0,
                2.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  -2.5
                ],
                [
                  0.0,
                  2.5
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                -1.0,
                -0.0
              ],
              [
                -2.0,
                -0.0
              ],
              [
                -3.0,
                -0.0
              ],
              [
                -4.0,
                -0.0
              ],
              [
                1.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Diagram",
          "anchor_choice": {
            "ThreeByThree": [
              "Center",
              "Top"
            ]
          },
          "texts": [
            {
              "content": "-4.0",
              "location": [
                -4.0,
                -2.5
              ]
            },
            {
              "content": "0.0",
              "location": [
                0.0,
                -2.5
              ]
            },
            {
              "content": "1.0",
              "location": [
                1.0,
                -2.5
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Diagram",
          "anchor_choice": {
            "ThreeByThree": [
              "Right",
              "Middle"
            ]
          },
          "texts": [
            {
              "content": "-2.0",
              "location": [
                -4.5,
                -2.0
              ]
            },
            {
              "content": "0.0",
              "location": [
                -4.5,
                0.0
              ]
            },
            {
              "content": "2.0",
              "location": [
                -4.5,
                2.0
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              -2.0615595959595963,
              1.2121212121212122
            ],
            [
              -1.839403110618951,
              1.3022649201555454
            ],
            [
              -1.5228769472013008,
              1.389818956883139
            ],
            [
              -1.0653527891330783,
              1.4320055052027012
            ],
            [
              -0.4426222847799641,
              1.2935624476900913
            ],
            [
              0.132579567859922,
              0.6568490418741022
            ],
            [
              -0.12898755628279623,
              -0.6132675517306243
            ],
            [
              -1.3863995418754707,
              -1.549641027223734
            ],
            [
              -2.4582512442631486,
              -1.6343680640471006
            ],
            [
              -3.0023844537427946,
              -1.381753488648064
            ],
            [
              -3.241176393198164,
              -1.1062065901626394
            ],
            [
              -3.3410802159694,
              -0.8790920180083739
            ],
            [
              -3.379098746867168,
              -0.7017543859649121
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Polyline": {
          "path": [
            [
              -2.0615595959595963,
              1.2121212121212122
            ],
            [
              -2.0148397104835065,
              1.2331256901326197
            ],
            [
              -1.964267514810803,
              1.2546147177062594
            ],
            [
              -1.9094219104821768,
              1.2764818310395536
            ],
            [
              -1.8498320177589564,
              1.298574727172742
            ],
            [
              -1.784973501410739,
              1.3206802890197975
            ],
            [
              -1.7142663162540506,
              1.3425048404216156
            ],
            [
              -1.6370751477864998,
              1.3636481495794415
            ],
            [
              -1.552714564833536,
              1.3835693045417283
            ],
            [
              -1.4604620189421265,
              1.4015421588441168
            ],
            [
              -1.3595834800972646,
              1.4165976725490252
            ],
            [
              -1.2493788861801456,
              1.4274503442906263
            ],
            [
              -1.1292578969968996,
              1.4324064336472881
            ],
            [
              -0.9988607545723481,
              1.4292535515678604
            ],
            [
              -0.8582440263106069,
              1.4151357623647325
            ],
            [
              -0.7081553055403892,
              1.386427726913234
            ],
            [
              -0.5504210995974134,
              1.3386386590207116
            ],
            [
              -0.3884608428155601,
              1.266405252218382
            ],
            [
              -0.22790434209683152,
              1.1636731531181619
            ],
            [
              -0.07721229636982406,
              1.0242108476332306
            ],
            [
              0.051933018711089254,
              0.8426188754953506
            ],
            [
              0.14486700699727056,
              0.6159293378587528
            ],
            [
              0.18531314221931658,
              0.3456529742494818
            ],
            [
              0.15821519943330645,
              0.03969711189480027
            ],
            [
              0.05372020992274863,
              -0.28687287711678994
            ],
            [
              -0.1289875562827963,
              -0.6132675517306244
            ],
            [
              -0.37992730485068993,
              -0.9167952563674376
            ],
            [
              -0.6799520627314016,
              -1.1778362487394478
            ],
            [
              -1.005294270294135,
              -1.3838398246099033
            ],
            [
              -1.332941071418675,
              -1.5307551215643718
            ],
            [
              -1.6446350662023768,
              -1.6218393957984598
            ],
            [
              -1.9285120968469383,
              -1.6650128543221876
            ],
            [
              -2.1787259479769583,
              -1.6701173001987604
            ],
            [
              -2.3940148278085145,
              -1.6468902121922016
            ],
            [
              -2.5760615122244186,
              -1.6038346280565094
            ],
            [
              -2.7281201039042426,
              -1.5478010843661232
            ],
            [
              -2.8540532635421627,
              -1.4840043466340604
            ],
            [
              -2.9577432410493185,
              -1.4162462484221823
            ],
            [
              -3.042778491665252,
              -1.3472000547430691
            ],
            [
              -3.1123186630779114,
              -1.278681748446529
            ],
            [
              -3.169063686023342,
              -1.2118783078089361
            ],
            [
              -3.215277397329565,
              -1.1475269367713905
            ],
            [
              -3.2528355110706713,
              -1.0860499022628578
            ],
            [
              -3.283280940512966,
              -1.0276531593470755
            ],
            [
              -3.307877691086461,
              -0.9723970894552236
            ],
            [
              -3.327659340648783,
              -0.9202465033240225
            ],
            [
              -3.3434707579595715,
              -0.871105570567874
            ],
            [
              -3.356003057832975,
              -0.8248419563097789
            ],
            [
              -3.3658224087163617,
              -0.7813033109734149
            ],
            [
              -3.373393531063705,
              -0.740328384536862
            ],
            [
              -3.379098746867168,
              -0.7017543859649121
            ]
          ],
          "path_choices": {
            "color": "Red"
          }
        }
      }
    }
  ]
}
